db/seeds/cip_seed.rb

```diff
@@ -1,18 +1,18 @@
-CourseYear.import([:id, :title, :content, :mentor_title], [
-  ["a4dc302c-ab71-4d7b-a10a-3116a778e8d5", "Your module materials", "<p></p>", "Coaching and mentoring support materials"],
-  ["793ad37c-09c1-49a4-922f-78b29be4bc87", "Your module materials", "TBC", "Coaching and mentoring support materials"],
-  ["fbf47b24-691b-4114-af34-6adcc9b45346", "Your module materials", "<p></p>", "Coaching and mentoring support materials"],
-  ["b61a7399-c503-4e37-81af-e45bdaddd93c", "Your module materials", "<p></p>", "Coaching and mentoring support materials"],
-  ["41a0ed38-4a04-436c-8559-2b7fc3b917c6", "Your module materials", "<p></p>", "Coaching and mentoring support materials"],
-  ["d7ac2099-81cb-4f95-9c32-ca68775c61d0", "Your module materials", "<p></p>", "Coaching and mentoring support materials"],
-  ["085b64e0-bd12-4795-a5b9-c3944af9d02e", "Your module materials", "TBC", "Coaching and mentoring support materials"],
-  ["441979d3-787c-42bd-9072-2309125ef709", "Your module materials", "<p></p>", "Coaching and mentoring support materials"]
-], on_duplicate_key_update: { conflict_target: [:id], columns: :all })
 CoreInductionProgramme.import([:id, :name, :course_year_one_id, :course_year_two_id, :slug], [
```

Comment:
Do the course_year_one_id and course_year_two_ids need to be in there, along with the nils below?

Reply:
I will remove them in a later PR. I think they don't need to be here, but I prefer to cut them when I cut them out of the database.